README.md

```diff
@@ -1,54 +1,164 @@
-## `Intl.RelativeTimeFormat` API Specification [draft]
+# `Intl.RelativeTimeFormat` API Specification [draft]
 
-### Status
+## Overview
+
+### Motivation
+
+Due to common use, relative time formatted values exist in a majority of websites and are available in the majority of the frameworks (e.g., React, via [react-intl](https://github.com/yahoo/react-intl) and [react-globalize](https://github.com/globalizejs/react-globalize); Ember, via [ember-intl](https://github.com/ember-intl/ember-intl)). Popular localization libraries like [momentjs](https://momentjs.com/), [formatjs](https://formatjs.io/), [globalizejs](http://globalizejs.com/), and [others](https://github.com/rxaviers/javascript-globalization) had implemented a formatting process for relative time values as well.
+
+It is highly probable that the majority of current relative time formatting implementations require a large portion of CLDR raw or compiled data to format relative time values. Bringing this into the platform will improve performance of the web and developer productivity as they no longer have to bring extra weight to format relative time values.
+
+### Usage
+
+```js
+// Create a relative time formatter in your locale.
+let rtf = new Intl.RelativeTimeFormat("en", {
+    style: "long", // "long" (default), "short", or "narrow"
+    type: "text" // "text" (default), or "numeric".
+});
+
+// Format relative time using the day unit.
+rtf.format(
+  -1,
+  "day" // "year", "quarter", "month", "week", "day", "sun", "mon", "tue", "wed", "thu", "fri", "sat", "hour", "minute", or "second".
```

Comment:
Technically, it doesn't change anything either to the spec or implementation. It would only add a couple of extra bytes (due to extra data) to the formatter.

If we remove it now, then add it later, would that present a challenge for developers to check support? This is, how would a developer know in advance whether or not these units are available?